services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py

```diff
@@ -99,32 +100,49 @@ async def wait_for_rabbitmq_liveness(app: FastAPI) -> None:
     )
 
 
+@lru_cache(maxsize=2)
```

Comment:
why 2?

Reply:
there are 2 clients at the moment: `rabbitmq_client` and `rabbitmq_rpc_server`